opensearch-operator/functionaltests/dummy_test.go

```diff
@@ -0,0 +1,31 @@
+package functionaltests
```

Comment:
didnt understood what the secret test is doing.

Reply:
It is just a minimal dummy test to quickly show how functional tests work without the complexity of an opensearch cluster.
The test does nothing more than apply the `dummy.yaml` file (as a `kubectl apply -f dummy.yaml` would do) and then verifies the secret is available in kubernetes.